vlsv_writer.h

```diff
@@ -92,7 +94,40 @@ namespace vlsv {
       bool writeWithReduction(const std::string& arrayName,const std::map<std::string,std::string>& attribs,
 			      const uint64_t& arraySize,T* array,MPI_Op operation);
    
+
```

Comment:
Please don't put code in header files, move 'getBuffer' and 'setBuffer' to vlsv_writer.cpp